docs/functions/function-crd.md

```diff
@@ -248,10 +248,10 @@ Function Mesh supports running Pulsar Functions in Java, Python and Go. This tab
 
 | Field | Description |
 | --- | --- |
-| `jarLocation` | The path to the JAR file for the function. It is only available for Pulsar functions written in Java. |
+| `jarLocation` | The path to the JAR file for the function. It is only available for Pulsar functions written in Java. If the package file uses the HTTP or HTTPS protocol, you need to set `enable-init-containers` to `true` when you install the Function Mesh Operator. For details about how to configure the Function Mesh Operator, see [Function Mesh Operator configurations](/reference/function-mesh-config.md).|
```

Comment:
it's not a requirement to use `init-containers` when the package file uses the HTTP or HTTPS protocol:

- if `enable-init-containers` is set to true, then when the package file uses HTTP/HTTPS, it will use wget to download the package
- if `enable-init-containers` is set to false, then when the package file uses HTTP/HTTPS, it will use pulsar-admin to download the package

the difference is that use `pulsar-admin` way required pulsar admin privilege while `wget` doesn't

the doc update should add a field: `DownloaderImage` in function-crd/source-crd/sink-crd, like below:

```
| --- | --- |
| `DownloaderImage` | The image used to download the package, which requires [pulsarctl](https://github.com/streamnative/pulsarctl) and `wget` installed, only takes effect when `enable-init-containers` is `true`. By default, an [official pulsarctl image](https://hub.docker.com/r/streamnative/pulsarctl) is used. |
```

Reply:
and in here: https://functionmesh.io/docs/reference/function-mesh-config, append below content to the `enable-init-containers` filed's description column:

```
Set it to true when you want to use `wget` instead of `pulsar-admin`(requires admin authentication of pulsar) to download packages from HTTP/HTTPS.
```